### PR TITLE
Update irrigation.jsonld

### PR DESCRIPTION
**Pull request by gasserluc**

### CODE REVIEW

File: irrigation/jsonld-contexts/irrigation.jsonld [hunk 1]
```diff
@@ -64,7 +65,9 @@
         "volumetricMoistureSaturation": "https://vocab.egm.io/volumetricMoistureSaturation",
         "waterConsumption": "https://vocab.egm.io/waterConsumption",
         "waterTension": "https://vocab.egm.io/waterTension",
+        "wateringIsEnalbed": "https://vocab.egm.io/wateringIsEnalbed",
```
Comment (bobeal) [inline irrigation/jsonld-contexts/irrigation.jsonld:68]:
```suggestion
        "wateringIsEnabled": "https://vocab.egm.io/wateringIsEnabled",
```

**Review by bobeal — COMMENTED** [review]
As a personal taste, I prefer having the verb in the first position (e.g. `isLimitationEnabled`), but well...

File: irrigation/jsonld-contexts/irrigation.jsonld (hunk 1 above)
Comment (gasserluc) [reply irrigation/jsonld-contexts/irrigation.jsonld]: Ok, I don't really mind, I have changed

File: irrigation/jsonld-contexts/irrigation.jsonld [hunk 2]
```diff
@@ -27,6 +27,8 @@
         "irrigationWeekday": "https://vocab.egm.io/irrigationWeekday",
         "irrigationDoseMinute": "https://vocab.egm.io/irrigationDoseMinute",
         "irrigationDailyDuration": "https://vocab.egm.io/irrigationDailyDuration",
+        "isWateringEnalbed": "https://vocab.egm.io/isWateringEnalbed",
+        "isWateringLimitationEnalbed": "https://vocab.egm.io/isWateringLimitationEnalbed",
```
Comment (bobeal) [inline irrigation/jsonld-contexts/irrigation.jsonld:31]: good but now, you have 2 `enalbed` 😆

Comment (gasserluc) [reply irrigation/jsonld-contexts/irrigation.jsonld:31]: Yes, we may want to disable the watering (i.e. no irrigation, we block the valve) 
and on the other hand we may want to disable the watering limitation (i.e. no limitation is applied we don't touch the valve) 

The normal case is isWateringEnalbed = true & isWateringLimitationEnalbed = true i.e, there is some irrigation AND we apply the limitation recommmended by the algorithm

Is that somewhat clear ?

**Review by bobeal — APPROVED** [review]

**Pull request merged**